src/vue/backend/src/views/GuidedSetup.vue

```diff
@@ -18,19 +18,57 @@
                         <idx-list-item>Bullet list item 2</idx-list-item>
                         <idx-list-item>Bullet list item 3</idx-list-item>
                     </idx-list>
-                    <idx-button tag="a" href="#" size="lg">Let's Get Started</idx-button>
+                    <idx-button href="#" size="lg" @click="() => openDialog()">Let's Get Started</idx-button>
                 </idx-block>
             </idx-block>
         </idx-container>
+        <idx-dialog :show="showDialog" @dismiss="closeDialog" customClass="gs-dialog">
```

Comment:
Each step of the set up will be its own link, so putting the card in the guided set up restricts its reusability. For this ticket, you only need the card by itself, and not the implementation in practice. When we build the individual pages, we can put the card in its corresponding page thereby making it adjust the content on a per-page basis.